manifests/l/LocalSend/LocalSend/1.15.0/LocalSend.LocalSend.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: LocalSend.LocalSend
+PackageVersion: 1.15.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+InstallerType: inno
+UpgradeBehavior: install

```

```suggestion
UpgradeBehavior: uninstallPrevious # MSIX version has been discontinued
```
This is needed to avoid Installer Type mismatch

---

Perhaps on the next version, this should be switched back to `install`?